### PR TITLE
New Monitr

### LINKED ISSUE #206: concept for improving monitr
it would be more useful if monitr functioned as a data logger that we can use to also add comments, highlight data, etc.
This could all be done on the filesystem level (not db backend, or so. maybe .md files for text. however, maybe monitor could have an internal cache for faster searches.)
We need to come up with a reasonable concept for what features we want.

### PR DESCRIPTION
**Pull request by pfafflabatuiuc**
A new version of Monitr with more features. For a guide on how to use see: https://toolsforexperiments-manual.readthedocs.io/en/latest/plottr/apps.html#monitr

### CODE REVIEW

**Comment by pfafflabatuiuc** [comment]
I have added watchdog as a testing requirement in test_requirements.txt. Should I add it as a requirement in requirements.txt and the setup file?

File: plottr/apps/json_veiwer.py [hunk 1]
```diff
@@ -0,0 +1,316 @@
+"""
```
Comment (wpfff) [inline plottr/apps/json_veiwer.py]: fix typo in filename

**Review by wpfff — CHANGES_REQUESTED** [review]
this works well, but please fix typos in filenames (and elsewhere if there are any ;)) and add watchdog to the requirements.

**Review by wpfff — APPROVED** [review]
looks good now.

**Pull request merged**